blog/_posts/2016-12-20-xyz-codemotion.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Code in Motion: Story of the XYZ team in Berlin
```

Comment:
The post title doesn't render properly if there is a colon (`:`) in there. I'd suggest to change it to something like `—` to avoid the problem :) 
Can you push the change?